recipes-extended/qemu-dm/qemu-dm/use-relative-xenstore-nodes.patch

```diff
@@ -1,104 +0,0 @@
-################################################################################
-SHORT DESCRIPTION:
-################################################################################
-Use relative paths instead of absolute ones for xenstore device model nodes.
```

Comment:
That seems necessary for stub-domains to handle these xenstore nodes?

Reply:
The commit message for the patch removing it explains that it's not used with our version of xen.

Reply:
Seems kind of out of the scope of the PR to me, since removing this doesn't help reduce the size of the stubdom. Also the patch remains valid, even if we don't currently use that code path... But I guess if we're absolutely sure we'll never run into this code ever again, then why not...

Reply:
@jandryuk Indeed, my apologies.

Reply:
@jean-edouard, yes, it's out of scope for the PR. I can drop it here and PR it separately tomorrow.

Reply:
@jandryuk @jean-edouard I don't think there is much to gain by having this PR'ed separately.

Reply:
@eric-ch @jean-edouard just let me know what you want me to do regarding the use-relative-xenstore-nodes.patch.

Sorry for putting the out-of-scope patch into this PR.

Reply:
It's fine doing it here, but again, please ensure that code path is dead forever.

Reply:
In `include/hw/xen/xen_common.h`
>#if CONFIG_XEN_CTRL_INTERFACE_VERSION < 41000
>#define XEN_COMPAT_PHYSMAP
 
In `hw/i386/xen/xen-hvm.c` all the `/local/domain/0` code is wrapped in
> #ifdef XEN_COMPAT_PHYSMAP
> static int xen_save_physmap(XenIOState *state, XenPhysmap *physmap)
> ... /local/domain/0 paths modified by patch
> #else 
> ... empty stub
> #endif 

Since we're past 4.10, we're not longer using those functions.  I think we are good.